Examples/UIExplorer/UIExplorerUnitTests/RCTUnicodeDecodeTests.m

```diff
@@ -0,0 +1,46 @@
+//
+//  RCTUnicodeDecodeTests.m
+//  UIExplorer
+//
+//  Created by Leo Natan (Wix) on 27/10/2016.
+//  Copyright © 2016 Facebook. All rights reserved.
```

Comment:
Ah, just copy it exactly from another file in the same directory. So in this case it should be:

/**
 * The examples provided by Facebook are for non-commercial testing and
 * evaluation purposes only.
 *
 * Facebook reserves all rights not expressly granted.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
 * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL
 * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
 * AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */